### DISCUSSION TITLE
Blueprint not connecting to panel.

### DISCUSSION THREAD

**Discussion by sambos76** [Blackymas/NSPanel_HA_Blueprint#792] [Q&A]
Hi, 

Thanks for all the work on this project.

I can't get the blueprint to update the panel.  I have reflashed and reinstalled everything a few times but still no luck.  I can see the panel in HA (it is online and I can interact with it) but the blueprint doesn't update it when I make changes.

Any help is appreciated.

Thanks

**Comment by edwardtfn** [comment]
I need a bit more information to be able to help..

1. Which version are you running?
1. Have you successfully uploaded the TFT file (`nspanel_eu.tft` or `nspanel_us.tft`)?
1. What can you see in your panel? (A picture can help here)
1. Please look for any entry related to this automation on Home Assistant logs (Settings > System > Logs).
1. Under your panel's device screen in Home Assistant, there is a sensor called "Nextion display" (under diagnostics). How is its state?

**Comment by Kipperking** [comment]
Hi,

I'm also having exactly this issue. Have the NSPanel showing and online in HA. Relays respond etc.  

The panel display is standard Ewelink but with no inputs so temp 0 and time are 00.00
Nextion display is noted as 'Unknown" in the ESPHome entity

I feel I successfully uploaded the nspanel_eu.tft, I've filled out the blueprint (with some basics to test) but nothing happens to the display when I press "update TFT display". It goes offline briefly in HA as if re-booting but there's no physical change on the screen and no update applied per the step through videos.

Thanks for your help in advance!

**Reply by edwardtfn** [reply]
Try this: https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/840#issuecomment-1564111149

**Reply by Kipperking** [reply]
Thanks, commenting on the other thread as I'm still struggling with the upload of the new code.

**Comment by augustoglauco** [comment]
Hi, I also having something like this. 
 - Got IP.
 - Screen stucked on ("Please Wait..." message)
- TFT: 3.3
 - ESPHome: 3.3
 - Blueprint: (nothing here)
 
- I had uninstalled/installed ESPHOME, deleted some files, Installed Blueprint by uploading file (nspanel_blueprint.yaml)  and using automatic upload method.
- "Update all" on ESPHome dashboard.  Seems to update ok, reboot.

The only weird thing happens when "Update TFT display" is pressed and  .tft file not changed, it starts uploading from 90%, but if i chage US->EU or UE->US, it's ok.
Thanks for your help in advance!

![img7](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/12896652/00f9a10d-4e79-4839-b1b3-088842d4775d)
![img5](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/12896652/0c4437c6-2e28-4683-9f0c-a0ca56e017a7)
![img4](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/12896652/ec416714-d445-44b7-8bf0-b7d74a634d9a)
![img3](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/12896652/7bbbe063-3228-429c-b9eb-5a5841d6dc62)
![img2](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/12896652/a0e46340-76fa-454c-ad8d-639a01c87631)
![img1](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/12896652/6df4d2b1-090c-424f-87bc-800704b694e5)
![img8](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/12896652/c59b6732-ce37-4457-a99a-580cb8763903)
![img9](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/12896652/64ea62da-df72-40a5-85e8-871964a3d331)

**Reply by edwardtfn** [reply]
Have you tried restarting Home Assistant?

**Reply by edwardtfn** [reply]
> The only weird thing happens when "Update TFT display" is pressed and .tft file not changed, it starts uploading from 90%, but if i chage US->EU or UE->US, it's ok.

I think this is normal with Nextion protocol.

**Reply by edwardtfn** [reply]
On [this post](https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/793#issuecomment-1537379872) I explain a bit about naming of service and entities created by this project, how the Blueprint look for those and some suggestions to fix a mismatch. If you have a mismatch, one of the consequences could be the system stuck on the boot screen, but in that case I would expect some error in your Home Assistant logs (Settings > System > Logs).
